### PR TITLE
Adapting partial code(file name start with A) to the sonar cloud rules

### PR DESCRIPTION
**Pull request by gabrywu**
Adapting partial code(file name start with A) to the sonar cloud rules

### CODE REVIEW

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/shell/AbstractShell.java [hunk 1]
```diff
@@ -281,7 +283,7 @@ public void run() {
         //Process has not terminated.
         //So check if it has completed 
         //if not just destroy it.
-        if (p != null && !shell.completed.get()) {
+        if (!shell.completed.get()) {
           shell.setTimedOut();
```
Comment (Jave-Chen) [inline dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/shell/AbstractShell.java]: If  runCommand is not called, process may  be null , use ` !(p  instanceof Process) ` to check it.

Comment (gabrywu) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/shell/AbstractShell.java]: process vaiable will be assigned before ShellTimeoutTimerTask, so no need to check

**Comment by dailidong** [comment]
> If runCommand is not called, process may be null , use !(p instanceof Process) to check it.

need recheck or update？

**Comment by dailidong** [comment]
please solve conflicting files ArrayUtils.java

**Pull request closed without merging**